wikimedia/wikipedia-ios
# T376069

**Phabricator:** 
https://phabricator.wikimedia.org/T376069

### Notes
* Still has slide in it - do not merge until cleared by Haley/Robin

### Test Steps
* Fresh install
1. In goal region/language, populate history with 6+ articles
2. Background and foreground app
3. Navigate to year in review
4. On the second slide that's usually the 1.5 billion slide, it should tell you what day of the week you've read most on if you've read more than 1 article on a given day

### Screenshots/Videos

![Simulator Screenshot - iPhone 15 Plus - 2024-11-20 at 09 44 55](https://github.com/user-attachments/assets/269231e1-8b0f-4ec7-845c-bf348b364121)
